godView.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<title> A-Team Frupal </title>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link type="text/css" rel="stylesheet" href="style.css"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" charset="utf-8"></script>
+    <script>
+        var SIZE = 25

```

Comment:
I just noticed that this initialization code is vestigial because the #mapView element is missing in the HTML below. This has been replaced by MapView.js. Oh, well.

Reply:
Removed that in the last update.